internal/status/status.go

```diff
@@ -136,26 +136,26 @@ func (s *Status) Details() []interface{} {
 	return details
 }
 
-// Error is an alias of a status proto. It implements error and Status,
-// and a nil Error should never be returned by this package.
-type Error spb.Status
+// ErrorT is an alias of a status proto. It implements error and Status,
+// and a nil *ErrorT should never be returned by this package.
+type ErrorT spb.Status
```

Comment:
Why is this necessary?

Reply:
If it's named `Error` and you embed it in another struct, then it makes things messy, because now there is a field named Error instead of a method named Error.  (Embedding syntax in Go is problematic.)